app/controllers/api/v1/my_osbb_controller.rb

```diff
@@ -0,0 +1,8 @@
+class Api::V1::MyOsbbController < ApplicationController
+  def balance
+    @balance = {
```

Comment:
jbuilder

Reply:
you should not create a hash in controller
but have some object as here you have current_user
then in jbuilder view use this object to create response

Reply:
Good?
![image](https://user-images.githubusercontent.com/31774981/100066230-796cfa80-2e3d-11eb-92c7-556a123febd8.png)